seq2science/rules/trackhub.smk

```diff
@@ -692,3 +692,9 @@ rule trackhub:
                         if line.startswith("compositeTrack"):
                             line = "autoScale group\n" + line
                         tf.write(line)
+
+            # make sure it readable for everyone (not writable)
+            for dirpath, dirnames, filenames in os.walk(output[0]):
+                os.chmod(dirpath, 0o755)
+                for filename in filenames:
+                    os.chmod(os.path.join(dirpath, filename), 0o755)
```

Comment:
Is this not more readable? (also fixes a typo in the comment)
```
# make the trackhub readable for everyone (not writable)
shell("chmod -R 755 {output[0]}")
```

Reply:
Yes, you are absolutely right! I dont like os.subprocess stuff, but shell of snakemake is so pretty